docs/.vscode/extensions.json

```diff
@@ -14,7 +14,7 @@
     // Tons of markdown goodies (lists, tables of content, so much more)
     "yzhang.markdown-all-in-one",
 
-    // [[wiki-links]], backlinking etc
+    // [[wikilinks]], backlinking etc
     "kortina.vscode-markdown-notes",
```

Comment:
Not for this PR, but we should remove this extension from the repo as well.

Reply:
https://github.com/foambubble/foam/pull/722